terraform/environments/electronic-monitoring-data/gateway_endpoint.tf

```
@@ -0,0 +1,10 @@
+
+resource "aws_vpc_endpoint" "gtw_ep_s3" {
+    vpc_id = data.aws_vpc.shared.id
+    service_name = "com.amazonaws.eu-west-2.s3"
+    route_table_ids = ["rtb-065e8c1f6a34c126a", "rtb-0a92da52c0aefb033"]
+
+    tags = {
+        Name = "S3_Gateway_Endpoint"
+    }
```

we should include the predefined `local.tags` as per the [moj technical guidelines](https://technical-guidance.service.justice.gov.uk/documentation/standards/documenting-infrastructure-owners.html#tags-you-should-use) also

```suggestion
    tags = merge(
        local.tags,
        {
            Name = "S3_Gateway_Endpoint"
        }
    )
```